src/tests/ftest/container/boundary.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python3
+"""
+  (C) Copyright 2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import time
+from apricot import TestWithServers
+from general_utils import DaosTestError
+from avocado.core.exceptions import TestFail
+from thread_manager import ThreadManager
+
+class BoundaryTest(TestWithServers):
+    """
+    Epic: Create system level tests that cover boundary tests and
+          functionality.
+    Testcase:
+          DAOS-8464: Test lots of pools and connections
+    Test Class Description:
+          Start DAOS servers, create pools and containers to the support limit.
+    :avocado: recursive
+    """
+
+    def create_container_and_test(self, pool=None, pool_num=1, cont_num=1, with_io=False):
+        """To create single container on pool.
+        Args:
+            pool (str): pool handle to create container.
+            pool_num (int): pool number.
+            container_num (int): container number to create.
+            with_io (bool): enable container test with execute_io.
+        """
+
+        try:
+            container = self.get_container(pool)
+        except (DaosTestError, TestFail) as err:
+            self.fail("#(3.{0}.{1}) container create failed. err={2}".format(
+                pool_num, cont_num, err))
+
+        self.log.info("===(3.{0}.{1})create_container_and_test, container {2} created..".format(
+            pool_num, cont_num, container))
+        if with_io:
+            io_run_time = self.params.get("run_time", '/run/container/execute_io/*')
+            rank = self.params.get("rank", '/run/container/execute_io/*')
+            obj_classs = self.params.get("obj_classs", '/run/container/execute_io/*')
+            try:
+                data_bytes = container.execute_io(io_run_time, rank, obj_classs)
+                self.log.info(
+                    "===(3.{0}.{1})Wrote {2} bytes to container {3}".format(pool_num, cont_num,
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions